readthedocs/search/tests/test_utils.py

```diff
@@ -66,3 +70,25 @@ def test_remove_only_one_version_index(self, api_client, all_projects):
         for project in ["pipeline", "docs"]:
             for version in [LATEST, STABLE]:
                 assert self.has_results(api_client, project, version)
+
+    @mock.patch("readthedocs.projects.tasks.search.reindex_version")
+    def test_index_project(self, reindex_version, all_projects):
```

Comment:
I tried to test this without mocking, but we don't have a good way to re-index things in tests... But at least I found a bug while trying to make it work.